src/benchUtil.c

```diff
@@ -566,36 +566,38 @@ char *taos_convert_datatype_to_string(int type) {
 }
 
 int taos_convert_string_to_datatype(char *type) {
-    if (0 == strcasecmp(type, "binary")) {
+    if (0 == strncasecmp(type, "binary", strlen("binary"))) {
```

Comment:
Doubt it leads a risk if a string like "binaryXXX" passed, but strncasecmp() will determine it's a valid type. Actually it should be unknown

Reply:
yeah, I do this on purpose, because fetch the row of the describe result from TDengine will give the data like 'INT\a', which is actually 'INT'.

Reply:
then you should use fetch_lengths() to cut the string with the right length

Reply:
ok, i will fix it